File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/EvaluateContextInstanceExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class EvaluateContextInstanceExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            var response = new ContextsApi(config).EvaluateContextInstance(
                projectKey: "projectKey_string",
                environmentKey: "environmentKey_string",
                requestBody: JsonSerializer.Deserialize<Dictionary<string, object>>("""
                    {
                        "key": "user-key-123abc",
                        "kind": "user",
                        "otherAttribute": "other attribute value"
                    }
                """)
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ContextsApi#EvaluateContextInstance: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/EvaluateContextInstanceExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ContextsApi(configuration).evaluateContextInstance(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  {
    "key": "user-key-123abc",
    "kind": "user",
    "otherAttribute": "other attribute value"
  }, // requestBody
  undefined, // limit
  undefined, // offset
  undefined, // sort
  undefined, // filter
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ContextsApi#evaluateContextInstance:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
